packages/@react-aria/utils/test/runAfterTransition.test.ts

```diff
@@ -0,0 +1,129 @@
+import {runAfterTransition} from '../src/runAfterTransition';
+
+class MockTransitionEvent extends Event {
+  propertyName: string;
+
+  constructor(type: string, eventInitDict?: TransitionEventInit) {
+    super(type, eventInitDict);
+    this.propertyName = eventInitDict?.propertyName || '';
+  }
+}
+
+describe('runAfterTransition', () => {
+  const originalTransitionEvent = global.TransitionEvent;
+  const nodes = new Set<Node>();
+
+  beforeAll(() => {
+    global.TransitionEvent = MockTransitionEvent as any;
+  });
+
+  afterAll(() => {
+    global.TransitionEvent = originalTransitionEvent;
+  });
+
+  beforeEach(() => {
+    jest
+      .spyOn(window, 'requestAnimationFrame')
+      // @ts-expect-error -- mock
+      .mockImplementation((cb) => cb());
```

Comment:
better to make use of `jest.useFakeTimers`
you'll need to add some
```
act(() => {jest.runOnlyPendingTimers();});
```
in order to trigger the mocked raf

Reply:
Updated.